File: scripts/gcap.py
#!/usr/bin/env python3
import os
import sys

cwd = os.getcwd()

# os.system(f'cd {cwd} && python -m unittest discover')

# filepath = os.path.join(cwd, 'requirements.py')
# if os.path.isfile(filepath):
#     os.system(f'cd {cwd} && python requirements.py')

os.system('git add .')
os.system(f'git commit --amend --no-edit')
os.system(f'git push -f')
os.system(f'git --no-pager log --oneline --decorate --graph --all -n 12')

File: scripts/gcap.js
#!/usr/bin/env node
const { runCommands } = require("./runCommands");

runCommands([
  'git add .',
  'git commit --amend --no-edit',
  'git push -f',
  'git --no-pager log --oneline --decorate --graph --all -n 12',
])